src/score_hv/stats_utils.py

```diff
@@ -0,0 +1,38 @@
+"""Copyright 2024 NOAA
+All rights reserved.
+
+Methods to calculate gridcell area weighted statistics
+The functions generally take the following two input variables:
+    xarray_variable - xarray variable data
+    gridcell_area_data - xarray variable containing gridcell areas, in the
+    same shape as xarray_variable
+"""
+
+import numpy as np
+
+def area_weighted_mean(xarray_variable, gridcell_area_weights):  
```

Comment:
Could we add some more information about the usage of this function and why we need to return the weighted averages separately from the calculation? I had to go look through the code usage to understand it given the existing comment. I see we have to calculate multiple weighted means first but that wasn't clear to me reading this function itself.

Reply:
Thanks for pointing out this valid concern. The benefit to having this function is it allows the assertion statements to run anytime the area weighted means are calculated without having to duplicate them for each (new) harvester.